create-ladder.js

```diff
@@ -26,4 +26,40 @@ const addHandlersToElements = () => {
     })(i);
   }
 }
+
+const addStep = document.getElementById('addStep');
```

Comment:
See your code [above](https://github.com/CYPIAPT-LNDSE/anxiety-ladder/pull/45/files#diff-cbc266616471666b17494f515a7f47a8R19), this is much better, since this is a function that is explicitly called onload, then goes and binds all the listeners.